src/main/java/com/exasol/adapter/dialects/mysql/MySqlSqlDialect.java

```diff
@@ -105,6 +104,11 @@ public NullSorting getDefaultNullSorting() {
         return NullSorting.NULLS_SORTED_AT_END;
     }
 
+    @Override
+    public String getStringLiteral(final String value) {
+        return super.quoteLiteralStringWithSingleQuote(value);
```

Comment:
```suggestion
        return super.quoteLiteralStringWithSingleQuote(value.replace("\\", "\\\\"));
```
https://dev.mysql.com/doc/refman/8.0/en/string-literals.html